package.json

```diff
@@ -22,7 +22,7 @@
   "main": "./lib/service.js",
   "scripts": {
     "start": "heroku-node-settings index.js",
-    "build:css": "mkdir -p public && sass src/main.scss public/main.css --style=compressed --load-path=bower_components --load-path=.",
+    "build:css": "mkdir -p public && sass src/main.scss  --style=compressed --load-path=bower_components --load-path=. | postcss --use autoprefixer --output public/main.css --map",
```

Comment:
do we have a browserslist config for the autoprefixer to use or are we using the default values?

Reply:
I've added IE11 to a browserlist, which is the main one

Reply:
although we never really test the registry in IE11, and I don't care to go to much extra effort for it for the registry